### ISSUE TITLE
Some apps has thicker title bar

### ISSUE THREAD

**Issue by NeuronSooup** [Fausto-Korpsvart/Gruvbox-GTK-Theme#40]
https://imgur.com/a/TZpIgu7

**Comment by Fausto-Korpsvart** [comment]
> https://imgur.com/a/TZpIgu7

Is imgur still working? Because the page has not been loading images for a while now.

Back on topic, I've been trying to fix the problem of rounded bottom corners in Firefox, that led me to have to play a bit with the thickness of the title bars; I'm still testing this, so in the next updates I'm going to try to make it so that the title bars can have the same thickness.

At least Firefox now has all four corners rounded xD
but I'll keep playing with this until the bars can be similar or the same.

![firefox-01](https://github.com/Fausto-Korpsvart/Gruvbox-GTK-Theme/assets/47998118/159d35a8-e2b9-4342-bd6a-f1f140893358)
![firefox-02](https://github.com/Fausto-Korpsvart/Gruvbox-GTK-Theme/assets/47998118/e8061d41-3e63-43bb-b4e2-5f387d51b337)

**Comment by NeuronSooup** [comment]
Firefox is fine though but the rest of the apps, I think it only happens to gtk 3 and apps like discord or spotify..
you can check the imgur again, it is working for me

**Comment by Fausto-Korpsvart** [comment]
> Firefox is fine though but the rest of the apps, I think it only happens to gtk 3 and apps like discord or spotify.. you can check the imgur again, it is working for me

I don't know what's going on with the imgur site, I can't see anything on that page on desktop, but I've been able to see your images on my phone; and yes, I understand, that's what I was referring to in the other comment.

I've been trying to fix the bug that many have reported about the rounded bottom corners in Firefox, so I've been playing around a bit until I found a way, not to fix it, but it helps to make the four corners look rounded, but it makes the bars grow a bit; so I've been trying to fix it by playing around a bit with the size of the bars until they're a bit uniform.

In my case I use Spotify Flatpak version and the bar doesn't look so thin but, as I said before, I'm going to keep trying until all the title bars look the same size, just give me some time while I figure it out. }:-]

**Comment by NeuronSooup** [comment]
Thank you and good luck, if you can I have a little suggesting that can you make the title bar just 1px less thic. I think gnome title bar is too thick in my opinion

**Comment by Fausto-Korpsvart** [comment]
> Thank you and good luck, if you can I have a little suggesting that can you make the title bar just 1px less thic. I think gnome title bar is too thick in my opinion

Greetings, @NeuronSooup.
Could you test the theme and tell me what the title bars look like now?

I think they look a bit similar now }:-]

**Comment by NeuronSooup** [comment]
Thank you, the issue has been fully resolved now. However, the BL version has no border 
https://imgur.com/a/gzs3MFg

**Comment by Fausto-Korpsvart** [comment]
> Thank you, the issue has been fully resolved now. However, the BL version has no border https://imgur.com/a/gzs3MFg

Actually it does have that little border, but the colour is too similar to the background colour, in the next update I will put a different colour to accentuate the border a bit more.

**Comment by NeuronSooup** [comment]
Thank you, a little of topic but I really admire your dedication to these projects. Maintaining a theme is hard already but you have even outdone that plus with shell theme and icons. I have been supporting these projects and will be for years and years to come.
Thank you for the amazing work you have done

**Comment by Fausto-Korpsvart** [comment]
> Thank you, a little of topic but I really admire your dedication to these projects. Maintaining a theme is hard already but you have even outdone that plus with shell theme and icons. I have been supporting these projects and will be for years and years to come. Thank you for the amazing work you have done

Greetings, @NeuronSooup , thanks for your comment.

Actually, it's funny, because I started these themes as something personal, you know, to combine my Neovim, Xmonad and Gnome configurations, and to be honest, when I created these repositories and published the themes on Pling, I had no expectations and didn't think many people would like them; but I started to receive feedback, and this is what made me continue with the themes as a project.

I'm glad that you all like them, and although sometimes, due to lack of time, I haven't updated the themes very often, but, to paraphrase your words: 'I will do it for years to come', as long as it's fun for me and useful for many people I will continue to maintain these projects.

Thank you very much for your support }:-]
